app/src/main/kotlin/org/fossify/dialer/fragments/ContactsFragment.kt

```diff
@@ -153,9 +153,9 @@ class ContactsFragment(context: Context, attributeSet: AttributeSet) : MyViewPag
         val filtered = allContacts.filter {
             getProperText(it.getNameToDisplay(), shouldNormalize).contains(fixedText, true) ||
                 getProperText(it.nickname, shouldNormalize).contains(fixedText, true) ||
-                it.phoneNumbers.any {
+                (fixedText.toDoubleOrNull() != null && it.phoneNumbers.any {
```

Comment:
Is there any reason why it's not `toIntOrNull()`? Phone numbers usually don't contain decimals.